lib/ms.js

```diff
@@ -21,6 +21,7 @@ function docpress (cwd, options) {
     .source('.')
     .destination(meta.dist)
     .metadata(meta)
+    .ignore('node_modules')
     .ignore(`!**/{${meta.docs}{,/**/*},*.md}`)
```

Comment:
I think my line of thinking was that this was enough to get node_modules ignored. I guess it wasn't!